charts/pg-db/templates/cluster.yaml

```diff
@@ -59,6 +59,14 @@ spec:
       accessmode: {{ .Values.pgPrimary.volumeSpec.accessmode }}
       storagetype: {{ .Values.pgPrimary.volumeSpec.storagetype }}
       storageclass: {{ .Values.pgPrimary.volumeSpec.storageclass }}
+    {{- if .Values.pgPrimary.annotations }}
+    annotations:
+      {{ toYaml .Values.pgPrimary.annotations | indent 4  }}
```

Comment:
Please have it indented to left like other toYaml usages, because I think when you use toYaml it makes a difference how you write it and indent then can be different.